### PR TITLE
Update create-validator.md

### PR DESCRIPTION
**Pull request by CharlesIVEE**
added rough draft docs for changes to deposit commands for rocketarb addon

### CODE REVIEW

File: docs/guides/node/create-validator.md [hunk 1]
```diff
@@ -5,29 +5,31 @@ The minipool handles a portion of your ETH, known as the **bond amount**, and a
...
+8-ETH bonded minipools are a feature of the [**Atlas**](../atlas/whats-new.md) upgrade and will not be available until Atlas has launched.
+::: 
 
 Before creating a minipool, the first step is to choose the amount of ETH you want to bond.
 With the Atlas update, Rocket Pool allows bond amounts of either **8 ETH** or **16 ETH**.
 This bond represents your literal stake in the game as a validator; if your node is penalized for poor performance or slashed for violating the rules of the protocol, the penalty is taken from your bond.
 
 In addition to your ETH bond, creating a validator also requires **supplemental collateral** in the form of the **RPL token**.
-The amount of RPL you need to stake depends on your bond size; the minimum is 10% of your _borrowed_ amount of ETH, and the maximum is 150% of your _bonded_ amount of ETH.
+The amount of RPL you need to stake depends on your bond size; the minimum is 10% of your *borrowed* amount of ETH, and the maximum is 150% of your *bonded* amound of ETH.
```
Comment (xrchz) [inline docs/guides/node/create-validator.md:20]: typo: amound

File: docs/guides/node/create-validator.md [hunk 2]
```diff
@@ -73,10 +76,11 @@ When you stake RPL, you do it for your **entire node**.
 This means you'll only need to handle the total RPL stake for your node if you plan to run multiple minipools.
 :::
 
+
```
Comment (xrchz) [inline docs/guides/node/create-validator.md:79]: some unnecessary blank lines and whitespace around here

File: docs/guides/node/create-validator.md [hunk 3]
```diff
@@ -5,29 +5,31 @@ The minipool handles a portion of your ETH, known as the **bond amount**, and a
...
 Thus, in order to create a validator using Rocket Pool, you need to **create a minipool**.
 
+
 ## Choosing a Bond Size
 
 ::: warning NOTE
-8-ETH bonded minipools are a feature of the [**Atlas**](../atlas/whats-new) upgrade, which was launched on April 17th 2023.
-:::
+8-ETH bonded minipools are a feature of the [**Atlas**](../atlas/whats-new.md) upgrade and will not be available until Atlas has launched.
+:::
```
Comment (jshufro) [inline docs/guides/node/create-validator.md:13]: trailing whitespace here

File: docs/guides/node/create-validator.md [hunk 4]
```diff
@@ -5,29 +5,31 @@ The minipool handles a portion of your ETH, known as the **bond amount**, and a
...
-The amount of RPL you need to stake depends on your bond size; the minimum is 10% of your _borrowed_ amount of ETH, and the maximum is 150% of your _bonded_ amount of ETH.
+The amount of RPL you need to stake depends on your bond size; the minimum is 10% of your *borrowed* amount of ETH, and the maximum is 150% of your *bonded* amound of ETH.
 
 If choosing **8 ETH**, you stake 8 of your own ETH and "borrow" 24 ETH from the staking pool to complete the validator.
 The minimum RPL stake is **2.4 ETH worth of RPL** (10% of 24), and the maximum is **12 ETH worth of RPL** (150% of 8).
 
 If choosing **16 ETH** you stake 16 of your own ETH and borrow 16 ETH from the staking pool to complete the validator.
 The minimum RPL stake is **1.6 ETH worth of RPL** (10% of 16), and the maximum is **24 ETH worth of RPL** (150% of 16).
 
+
```
Comment (jshufro) [inline docs/guides/node/create-validator.md:28]: remove extra newline

File: docs/guides/node/create-validator.md [hunk 5]
```diff
@@ -173,7 +185,7 @@ Please choose an amount of RPL to stake:
 Select how much you'd like to stake, then confirm the operation.
 
 The first time you run this command, it will involve two transactions - one to **approve** the Rocket Pool staking contract to access your RPL, and one to **stake** your RPL with it.
-Subsequent runs will only require the **stake** transaction, since the token has already been approved.
+Subsequent runs will only require the **stake** transaction, since the token has already been approved.
```
Comment (jshufro) [inline docs/guides/node/create-validator.md:188]: trailing whitespace here

File: docs/guides/node/create-validator.md [hunk 6]
```diff
@@ -253,15 +264,16 @@ If you change any of those variables, the minipool address for a given salt will
 
 For more advanced usage (such as searching for a different node address or changing how many CPU cores are used for searching), take a look at the help text with `rocketpool minipool find-vanity-address --help`.
 
+
```
Comment (jshufro) [inline docs/guides/node/create-validator.md:267]: remove extra newline

File: docs/guides/node/create-validator.md [hunk 7]
```diff
@@ -434,7 +473,7 @@ Expected rewards:     16.010614 ETH
 Use latest delegate:  no
 Delegate address:     <your delegate address>
 Rollback delegate:    <none>
-Effective delegate:   <your delegate address>
+Effective delegate:   <your delegate address>
```
Comment (jshufro) [inline docs/guides/node/create-validator.md:476]: trailing whitespace

File: docs/guides/node/create-validator.md [hunk 8]
```diff
@@ -449,15 +488,17 @@ You have officially created a validator with Rocket Pool!
 
 Have a look at the next sections in Monitoring and Maintenance to learn how to watch your validator's performance and health over time.
 
+
```
Comment (jshufro) [inline docs/guides/node/create-validator.md:491]: remove extra newline

File: docs/guides/node/create-validator.md [hunk 9]
```diff
@@ -449,15 +488,17 @@ You have officially created a validator with Rocket Pool!
...
 
 Conveniently, your Rocket Pool node is capable of hosting as many minipools as you want.
 You **do not** need to create a new node for each minipool.
 
 If you would like to make a second (or third, or fourth...) minipool for your node, all you need to do is run `rocketpool node deposit` again.
-Note that you may need to stake more RPL first to maintain an overall collateral level of at least 10% of your _borrowed_ ETH before you do this.
+Note that you may need to stake more RPL first to maintain an overall collateral level of at least 10% of your *borrowed* ETH before you do this.
 Also, you won't be able to reuse an old vanity address salt - you'll need to search for another unique one for each of your minipools.
 
+
```
Comment (jshufro) [inline docs/guides/node/create-validator.md:501]: remove extra newline

**Comment by nickdoherty** [comment]
Closing this due to inactivity, feel free to reopen with the above recommended fixes.

**Comment by CharlesIVEE** [comment]
Nick

Thanks for the heads up — were the changes shown just comments and not
edits? They appeared to be on sections other than what I was working on
personally.

I appreciate your help
~Joe

**Pull request closed without merging**